src/main/kotlin/com/ampnet/auditornode/persistence/model/AssetCategoryId.kt

```diff
@@ -0,0 +1,5 @@
+package com.ampnet.auditornode.persistence.model
+
+import java.math.BigInteger
+
+inline class AssetCategoryId(val value: BigInteger) // TODO use `value class` instead on later Kotlin version
```

Comment:
You can use `Int` if you don't any special function from `BigIntger`.

Reply:
Generated contract API returns `BigInteger`. Using `Int` here would mean that it is possible to get an overflow.

Reply:
I get it, my bad.